{"uuid": "ffcfa1c5-54f0-49fd-83fb-506c44751413", "children": ["0beb682f-f8a8-4cf9-b1b1-8c8d2513d2a1"], "befores": [{"name": "Conftest", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'get'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 783, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/liuchengxu/work/sunxing/sunxing/TestSuite/CcsUserLoginController/conftest.py\", line 29, in Conftest\n    token = response1.get('msg')\n"}, "start": 1582534809076, "stop": 1582534809263}], "start": 1582534809076, "stop": 1582534809334}